website/docs/d/cos_bucket.html.markdown

```diff
@@ -35,6 +36,31 @@ output "bucket_private_endpoint" {
 }
 ```
 
+# cos satellite bucket
+
+Create a cos satellite bucket. See the architecture https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-about-cos-satellite for more details. We are using existing cos instance to create bucket , so no need to create any cos instance via a terraform. Cos satellite does not support all features see the section **What features are currently supported?** in https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-about-cos-satellite.
+IBM Satellite documentation https://cloud.ibm.com/docs/satellite?topic=satellite-getting-started . We are supporting object versioning and expiration features as of now. Firewall is not supported yet.
+
+## Example usage
+
+```terraform
```

Comment:
This doc is for datasource please provide a valid example to read a bucket for satellite based COS